pkg/interop/native/gas/gas.go

```diff
@@ -10,12 +10,12 @@ const Hash = "\x28\xb3\xad\xab\x72\x69\xf9\xc2\x18\x1d\xb3\xcb\x74\x1e\xbf\x55\x
 
 // Symbol represents `symbol` method of GAS native contract.
 func Symbol() string {
-	return contract.Call(interop.Hash160(Hash), "symbol", contract.ReadStates).(string)
+	return contract.Call(interop.Hash160(Hash), "symbol", contract.NoneFlag).(string)
```

Comment:
Should be a part of 621d5985282da510866b8f970269888c71cd7c44?

Reply:
Fixed.